filament/backend/src/opengl/platforms/PlatformEGLAndroid.cpp

```diff
@@ -557,6 +563,12 @@ bool PlatformEGLAndroid::convertSyncToFd(Sync* sync, int* fd) noexcept {
     }
 
     SyncEGLAndroid const& eglSync = static_cast<SyncEGLAndroid&>(*sync);
+    if (eglSync.sync == EGL_NO_SYNC_KHR) {
+        LOG(ERROR) << "Cannot convert sync to fd, because the underlying fence "
+                      "does not exist - does this display support fences?";
```

Comment:
I would shorten this message. If `ANDROID_native_fence_sync` is supported, this should not fail here.
so I would do something like:
```
if (sync == NO_SYNC) {
 if (ANDROID_native_fence_sync)
   LOG(ERROR) << "invalid fence"
 else
   LOG(WARNING) << "fences not supported"
}
```